docs/releases/1.8.txt

```diff
@@ -388,6 +388,10 @@ Generic Views
   require a ``form_class`` to be provided anymore. If not provided ``form_class``
   defaults to :meth:`~django.views.generic.edit.FormMixin.get_form_class()`.
 
+* Any placeholders in :attr:`~django.views.generic.edit.ModelFormMixin.success_url`
```

Comment:
Placeholders in :attr:`ModelFormMixin.success_url <django.views.generic.edit.ModelFormMixin.success_url>` now support the ...

Note about deprecation should technically appear under "Features Deprecated".